spec/features/bidder_interacts_with_auction_spec.rb

```diff
@@ -85,4 +113,83 @@
     expect(page).to have_content("No bids yet.")
   end
 
+  scenario "Viewing bid history for running auction" do
+    Timecop.scale(36000) do
+      create_running_auction
+    end
+    path = auction_bids_path(@auction.id)
+    visit path
+
+    # sort the bids so that newest is first
+    bids = @auction.bids.sort_by {|bid| bid.created_at}.reverse
+
+    # ensure the table has the correct content, in the correct order
+    bids.each_with_index do |bid, i|
+      row_number = i + 1
+      unredacted_bidder_name = bid.bidder.name
+      bid = Presenter::Bid.new(bid)
+
+      # check the "name" column
+      within(:xpath, cel_xpath(row_number, 1)) do
```

Comment:
I did have something like that! I got rid of it when I realized I'd also need `expect_content_not_in_row_number`. And then I thought, I could pass a block to a single function where the expectation could be called. It felt a little bit too close to reinventing parts of RSpec. Currently, a new set of :eyes: could see `within` and Google "rspec" within, and know immediately how the code here works.